samza-api/src/main/java/org/apache/samza/application/TaskApplication.java

```diff
@@ -50,9 +50,10 @@
  *     KafkaOutputDescriptor<PageViewEvent>> outputStreamDescriptor =
  *         trackingSystemDescriptor.getOutputDescriptor("recentPageViewEvent", new JsonSerdeV2<>(PageViewEvent.class)));
  *
- *     appDescriptor.addInputStream(inputStreamDescriptor);
- *     appDescriptor.addOutputStream(outputStreamDescriptor);
- *     appDescriptor.setTaskFactory((StreamTaskFactory) () -> new PageViewTask());
+ *     appDescriptor
+ *         .addInputStream(inputStreamDescriptor)
```

Comment:
Can we change the comments here to withXXX().withXXX()?

Reply:
👍 Will change them ti withInput/OutputStream.